recipes/libgettext/all/conanfile.py

```diff
@@ -56,15 +56,14 @@ def config_options(self):
         if self.settings.os == "Windows":
             self.options.rm_safe("fPIC")
 
+        self.options.threads = {"Solaris": "solaris", "Windows": "windows"}.get(str(self.settings.os), "posix")
```

Comment:
does this need validate logic? I'd image passing on windows threads on Ubuntu would fail?